### DISCUSSION TITLE
Embed Elsa designer in winform or wpf

### DISCUSSION THREAD

**Discussion by mbandu** [elsa-workflows/elsa-core#576] [Q&A]
Is that possible to embed Elsa designer in winform or wpf?

thanks

**Comment by adsgreen** [comment]
You might be able to use a WebView/WebView2 component to host the page?
Not tried or tested it though

**Reply by sfmskywalker** [reply]
Indeed. Since the designer is a web component, any host that is capable of executing HTML would be capable of hosting the designer.
